core/arch/arch_exports.h

```diff
@@ -609,6 +609,10 @@ cleanup_and_terminate(dcontext_t *dcontext, int sysnum, ptr_uint_t sys_arg1,
 
 bool
 cpuid_supported(void);
+#ifdef AARCHXX
+bool
+mrs_id_reg_supported(void);
```

Comment:
This is only used inside arch/ so no need to be in arch_exports.h: can be in arch.h

Reply:
Done.